CONTRIBUTING.md

```diff
@@ -179,5 +179,5 @@ Please sign our [Contributor License Agreement (CLA)][cla] before sending pull r
 [repo]: https://github.com/hypertrons/hypertrons-crx
 [angular-commit-message-format]: https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines
 [pr-template]: ./.github/pull_request_template.md
-[issue-template]: ./.github/issue_template.md
+[issue-template]: https://github.com/hypertrons/hypertrons-crx/issues/673
```

Comment:
Hi @andyhuang18, the link for issue-template should be this one: https://github.com/hypertrons/hypertrons-crx/tree/master/.github/ISSUE_TEMPLATE

and another suggestion:

<img width="1692" alt="image" src="https://github.com/hypertrons/hypertrons-crx/assets/32434520/596f5fc9-be64-4043-be93-6c4934d9cd2b">

Reply:
Copy that